sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -856,6 +883,7 @@ public synchronized void shutdown() {
             public void run() {
                 mScheduled = false;
                 triggerIngestion(GroupState.this);
+                SharedPreferencesManager.remove(START_TIMER + mName);
```

Comment:
We don't know whether the call to the server side is successful or not. 
So in this implementation, if the call isn't successful and we haven't reached batchSizeLimit to the moment, the next retry will be delayed as everything is ok.
I suggest clearing timestamp after the http call succeeds.

Reply:
@russelarms It doesn't matter. The timer has been finished.